guacamole/src/main/webapp/translations/ca.json

```diff
@@ -0,0 +1,1004 @@
+{
+    
+    "NAME" : "Catalan",
+    
+    "APP" : {
+
+
+
+
```

Comment:
Looks like a few extra spaces, here.  Just one should be enough :-).

Reply:
yes, one is sufficient. I removed the extra lines